addons/crm/views/crm_lead_views.xml

```diff
@@ -173,20 +173,13 @@
                                     <i class="fa fa-ban" style="color: red;" role="img" title="This email is blacklisted for mass mailing"
                                         aria-label="Blacklisted" attrs="{'invisible': ['|', ('is_blacklisted', '=', False), ('partner_address_email', '!=', False)]}" groups="base.group_user"/>
                                     <field name="email_from"
-                                        attrs="{'invisible': [('partner_address_email', '!=', False)]}" string="Email" widget="email"/>
+                                        attrs="{'invisible': [('partner_id', '!=', False)]}" string="Email" widget="email"/>
                                     <i class="fa fa-ban" style="color: red;" role="img" title="This email is blacklisted for mass mailing"
                                         aria-label="Blacklisted" attrs="{'invisible': ['|', ('partner_is_blacklisted', '=', False), ('partner_address_email', '=', False)]}" groups="base.group_user"/>
                                     <field name="partner_address_email"
-                                        attrs="{'invisible': [('partner_address_email', '==', False)]}" widget="email" string="Email"/>
+                                        attrs="{'invisible': [('partner_id', '==', False)]}" widget="email" string="Email"/>
```

Comment:
'==' ? Does that work ? Should probably be '=' like all other domains :) .

Reply:
It seemed to work but I've fixed it.